tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -355,6 +355,24 @@ When wrapping a conditional expression over multiple lines, the ``and``/``or``
 keywords should be placed at the beginning of the line continuation, not at the
 end of the previous line.
 
+Parentheses:
+
+::
+
+    if ($Sprite.animation == "back_crouch"
+            or $Sprite.animation == "front_crouch"
+            or $Sprite.animation == "horizontal_crouch"):
+        do_something()
```

Comment:
```gdscript
    if ($Sprite.animation == "back_crouch"
            or $Sprite.animation == "front_crouch"
            or $Sprite.animation == "horizontal_crouch"
    ):
        do_something()
```

or

```gdscript
    if (
            $Sprite.animation == "back_crouch"
            or $Sprite.animation == "front_crouch"
            or $Sprite.animation == "horizontal_crouch"
    ):
        do_something()
```

or

```gdscript
    if (
        $Sprite.animation == "back_crouch"
        or $Sprite.animation == "front_crouch"
        or $Sprite.animation == "horizontal_crouch"
    ):
        do_something()
```

?

Reply:
I tend to use the second one personally, as it's what the GDScript style guide recommends.